guides/TrainingModelsHPC.md

```diff
@@ -0,0 +1,169 @@
+# Train a detector in the cluster
+
+0.  **Prep**
+
+    - If not in the SWC network: connect to the SWC VPN.
+
+1.  **Connect to the SWC HPC cluster**
+
+    ```
+    ssh hpc-gw1
+    ```
+
+    It may ask for your password twice. To set up SSH keys for the SWC cluster, see [this guide](https://howto.neuroinformatics.dev/programming/SSH-SWC-cluster.html#ssh-keys).
+
```

Comment:
Should we add line to git clone and setup env as the first step, or maybe prep step?

Reply:
We don't need to do that because the bash script installs the crabs package from git